setuptools_rust/version.py

```diff
@@ -0,0 +1,4 @@
+__version__ = version = "1.4.1"
+__version_tuple__ = version_tuple = tuple(
+    map(lambda x: int(x[1]) if x[0] < 3 else x[1], enumerate(__version__.split(".")))
```

Comment:
This `__version_tuple__` is for compatibility with previous `setuptools-scm`, it's kinda ugly, we could remove it in 2.0

Reply:
👍 let's note it in #259